man/man8/ipa-healthcheck.8

```diff
@@ -37,7 +37,7 @@ Execute one or more checks within this given source.
 Execute this particular check within a source. A source must be supplied as well with this option.
 .TP
 \fB\-\-output\-type\fR=\fITYPE\fR
-Set the output type. The default is JSON.
+Set the output type. Supported variants are \fBhuman\fR and \fBjson\fR. The default is JSON.
```

Comment:
The default should be lower-case json, not JSON. I just double-checked this. It is case sensitive.

Reply:
Yep, this was the original sentence. I fixed it now.